addons/purchase/static/src/xml/purchase_dashboard.xml

```diff
@@ -1,68 +1,62 @@
+                            <a href="#" class="btn btn-light align-items-center d-flex w-100 h-100 p-0 border-0 bg-100 font-weight-normal "><div class="w-100 p-2" t-out="values['my_late']"/></a>
+                        </div>
+                </div>
+                <div class="g-col-6 h-100">
+                            <div class="grid h-100 gap-1">
+                                <div class="g-col-3 py-2 d-flex align-items-center">Avg Order Value (<t t-out="values['company_currency_symbol']"/>)</div>
+                                <div class="g-col-3 py-2 bg-light d-flex align-items-center justify-content-center"><span><t t-out="values['all_avg_order_value']"/></span></div>
+                                <div class="g-col-3 d-flex align-items-center">Purchased Last 7 Days (<t t-out="values['company_currency_symbol']"/>)</div>
+                                <div class="g-col-3 py-2 bg-light d-flex align-items-center justify-content-center"><span><t t-out="values['all_total_last_7_days']"/></span></div>
 
-            <div class="col-sm-7">
-            <table class="table table-sm">
-                <!-- thead needed to avoid list view rendering error for some reason -->
-                <thead>
-                    <tr>
-                        <!-- can't use th tag due to list rendering error when no values in list... -->
-                        <td class="o_text">Avg Order Value (<t t-esc="values['company_currency_symbol']"/>)</td>
-                        <td><span><t t-esc="values['all_avg_order_value']"/></span></td>
-                        <td class="o_text">Purchased Last 7 Days (<t t-esc="values['company_currency_symbol']"/>)</td>
-                        <td><span><t t-esc="values['all_total_last_7_days']"/></span></td>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <td class="o_text">Lead Time to Purchase</td>
-                        <td><span><t t-esc="values['all_avg_days_to_purchase']"/> &#160;Days</span></td>
-                        <td class="o_text">RFQs Sent Last 7 Days</td>
-                        <td><span><t t-esc="values['all_sent_rfqs']"/></span></td>
-                    </tr>
-                </tbody>
-            </table></div>
-        </div></div>
+                                <div class="g-col-3 py-2 d-flex align-items-center">Lead Time to Purchase</div>
+                                <div class="g-col-3 py-2 text-center bg-light"><span><t t-out="values['all_avg_days_to_purchase']"/> &#160;Days</span></div>
+                                <div class="g-col-3 py-2 d-flex align-items-center">RFQs Sent Last 7 Days</div>
+                                <div class="g-col-3 py-2 text-center bg-light "><span><t t-out="values['all_sent_rfqs']"/></span></div>
+                            </div>
+                </div>
+            </div>
+        </div>
     </t>
 </templates>
+
+            
```

Comment:
Add a new line at the end of each file... there are editor's extensions that hi-lights these kind of issues ;)